test/lit/passes/post-emscripten.wast

```diff
@@ -40,4 +47,11 @@
  (export "__start_foo" (global $foo_start))
  ;; CHECK:      (export "__stop_foo" (global $foo_stop))
  (export "__stop_foo" (global $foo_stop))
+ (func $meminit
+  (memory.init $mem $data2
+   (i32.const 3000000000)
+   (i32.const 0)
+   (i32.const 21)
+  )
+ )
```

Comment:
Why does this function not appear in the CHECKs?

Reply:
Oh.. i need to re-run the auto-generater.